eclass/distutils-r1.eclass

```diff
@@ -204,7 +204,7 @@ esac
 # This is an optimization that can avoid the overhead of calling into
 # the build system in pure Python packages and packages using the stable
 # Python ABI.
-DISTUTILS_ALLOW_WHEEL_REUSE=1
+: ${DISTUTILS_ALLOW_WHEEL_REUSE=1}
```

Comment:
This I think is missing a `:`:
```
: ${DISTUTILS_ALLOW_WHEEL_REUSE:=1}
```

Reply:
Both with and without a `:` are valid bash and mean two different things.

- with the `:` -- if `DISTUTILS_ALLOW_WHEEL_REUSE=` is set and blanked in the environment, it falls back to 1
- without the `:` -- if `DISTUTILS_ALLOW_WHEEL_REUSE=` is set and blanked in the environment, it is still *set* so we do nothing, and simply respect the value in the environment.

Reply:
Since later on we check whether the value is non-empty, it is utterly pointless to add `:` because then any attempt to override it to empty would trigger falling back to the eclass value. Supporting "set it in the environment to a blank value" is an intentional design choice.